scripts/projects.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+
+var allProjects = [];
+
+function Projects(cat) {
+  this.name = cat.name;
+  this.image = cat.image;
+  this.description = cat.description;
+  this.date = cat.date;
+  this.link = cat.link;
+}
+
+Projects.prototype.toHtml = function() {
+  var $newProject = $('summary.template').clone().removeClass('template');
+
+  $newProject.find('address').text('this.name');
```

Comment:
Why do you have the name of your project inside an `address` tag? Change this here and in your HTML file so the tag is semantically appropriate.